src/test/java/com/scriptbasic/executors/operators/TestJavaAccess.java

```diff
@@ -52,15 +52,15 @@ public void canUseOverloadedMethodInsecureMode() throws Exception {
 
     @Test()
     public void cannotUseOverloadedMethodSecureMode() {
-        Assertions.assertThrows(BasicRuntimeException.class, () ->
+        Assertions.assertThrows(ScriptBasicException.class, () ->
```

Comment:
Could you help me to understand why this change is needed? `BasicRuntimeException` is extending `ScriptBasicException`. Why to check a broader assertion when the more specific is OK?